### PR TITLE
Navigation Feature #166

### PR DESCRIPTION
**Pull request by sseff**
I have created navigation part

Capture
.

### CODE REVIEW

**Comment by CenekSanzak** [comment]
This branch is forked from the branch frontend and it should be merged to frontend branch. So, I changed the base branch.

File: frontend/package.json [hunk 1]
```diff
@@ -25,9 +25,11 @@
     "@babel/runtime": "^7.16.0",
     "@emotion/react": "^11.5.0",
     "@emotion/styled": "^11.3.0",
+    "@material-ui/icons": "^4.11.2",
     "@mui/icons-material": "^5.1.0",
     "@mui/material": "^5.0.6",
     "@mui/styles": "^5.1.0",
+    "clsx": "^1.1.1",
```
Comment (CenekSanzak) [inline frontend/package.json:32]: I see that clsx is not used in the project. Unused dependencies should be removed.

File: frontend/src/Views/Shared/Header.js [hunk 2]
```diff
@@ -1,30 +1,180 @@
...
+            width: '12ch',
+            '&:focus': {
+                width: '20ch',
+            },
+        },
+    },
+}));
+
+
+const useStyles = makeStyles(theme => createStyles({
```
Comment (CenekSanzak) [inline frontend/src/Views/Shared/Header.js:64]: These styles are not used in the function. It is just initialized in the line 95, you should remove the unused line for a clear and readable code.

**Review by CenekSanzak — CHANGES_REQUESTED** [review]
You created a good looking and user friendly ui for the header. I think the styles are good and the navigation is efficient. Also it gives the opportunity to add new links. 
I requested some changes as comments to some code lines. 
Also there a few minor requests. I think it is unnecessary to use a hr element belove the header. And I think you should add some margin between buttons.

**Comment by sseff** [comment]
Thanks for reviews, I changed them.

**Pull request merged**